src/pytest/test_spell_check.py

```diff
@@ -20,6 +20,12 @@ def testDictDelete():
     env.expect('ft.dictdel', 'dict', 'term3').equal(1)
     env.expect('keys', '*').equal([])
 
+def testDictDeleteOnFlush():
+    env = Env()
+    env.expect('ft.dictadd', 'dict', 'term1', 'term2', 'term3').equal(3)
+    env.expect('FLUSHALL').equal(True)
+    env.expect('ft.dictadd', 'dict', 'term4', 'term5', 'term6').equal(3)
```

Comment:
We can make sure dictionary is empty.
Otherwise, looks great!

Reply:
Done